ngxunprivinst.sh

```diff
@@ -362,6 +366,12 @@ upgrade() {
         [ -d $TMPDIR/usr/lib/ ] && cp -a $TMPDIR/usr/lib/* $ABSPATH/usr/lib/
         [ -d $TMPDIR/usr/lib64/ ] && cp -a $TMPDIR/usr/lib64/* $ABSPATH/usr/lib64/
         check_modules_deps
+        TARGETVER=$($ABSPATH/usr/sbin/nginx -v 2>&1 | cut -d '(' -f 2 | cut -d ')' -f 1 | cut -d'-' -f 3 | tr -d 'r')
+        if [ $TARGETVER -ge 31 ]; then
+            if ! $ABSPATH/usr/sbin/nginx -p $ABSPATH/etc/nginx -c nginx.conf -T 2>&1 | grep 'uuid_file' | grep -vE '^(.*)#.*uuid_file' >/dev/null; then
```

Comment:
This approach won't work if nginx.conf already contains mgmt{} block, without `uuid_file` directive, but this isn't the case for unpriv type of installation as I understand.

Reply:
Indeed, this is on "install" phase of the script where it is guaranteed nginx.conf is not edited by the user yet.

Reply:
Not really, this is an upgrade phase, so purely theoretically the user could end up with a non-working config with 2 mgmt blocks.

Reply:
Ah yes, I was confused - assumed github will show comments based on their appearance in the diff file.

I'll see if we can easily detect both cases.  But in reality I guess yeah, anyone who's going to be running own nginx.conf will still need to provide uuid_file directive to get rid of the annoying warnings anyway.